spec/models/vote_spec.rb

```diff
@@ -16,81 +16,172 @@
+    end
   end
 
   describe ".process_vote" do
-   it "throws an error on invalid params" do
-    expect(Vote.process_vote(subject_class, invalid_subject_id, invalid_user, value)).to be nil
-   end
+    it "throws an error on invalid params" do
+      expect(Vote.process_vote(subject_class, invalid_subject_id, invalid_user, value)).to be nil
+    end
   end
 
   describe ".total_votes" do
-   it "returns an integer value" do
-    allow(Vote).to receive(:subject_of_vote).and_return (subject)
-    expect(Vote.total_votes(subject, subject_id)).to be_an Integer
-    expect(Vote.total_votes(subject, subject_id)).to eq(0)
-   end
+    it "returns an integer value" do
+      allow(Vote).to receive(:subject_of_vote).and_return (subject)
+      expect(Vote.total_votes(subject, subject_id)).to be_an Integer
+      expect(Vote.total_votes(subject, subject_id)).to eq(0)
+    end
   end
 
   describe ".subject_of_vote" do
-   it "returns the object the vote belongs to" do
-    expect(Vote.subject_of_vote(subject_class, subject_id)).to be_a Question
-    expect(Vote.subject_of_vote(subject_class, subject_id)).not_to be_an Answer
-    expect(Vote.subject_of_vote(subject_class, subject_id)).not_to be_a Comment
-   end
+    it "returns the object the vote belongs to" do
+      expect(Vote.subject_of_vote(subject_class, subject_id)).to be_a Question
+      expect(Vote.subject_of_vote(subject_class, subject_id)).not_to be_an Answer
+      expect(Vote.subject_of_vote(subject_class, subject_id)).not_to be_a Comment
+    end
   end
 
   describe ".subject_exists" do
-   it "returns boolean depending on the existence of the subject" do
-    expect(Vote.subject_exists?(subject_class, subject_id)).to eq(true)
-    expect(Vote.subject_exists?(subject_class, invalid_subject_id)).to eq(false)
-   end
+    it "returns boolean depending on the existence of the subject" do
+      expect(Vote.subject_exists?(subject_class, subject_id)).to eq(true)
+      expect(Vote.subject_exists?(subject_class, invalid_subject_id)).to eq(false)
+    end
   end
 
   describe ".voted?" do
-   it "returns false if the user has not voted" do
-    expect(Vote.voted?(subject_class, subject_id, user)).to eq(false)
-    expect(Vote.voted?(subject_class, subject_id, user, invalid_value)).to eq(false)
-    expect(Vote.voted?(subject_class, invalid_subject_id, user, value)).to eq(false)
-   end
+    it "returns false if the user has not voted" do
+      expect(Vote.voted?(subject_class, subject_id, user)).to eq(false)
+      expect(Vote.voted?(subject_class, subject_id, user, invalid_value)).to eq(false)
+      expect(Vote.voted?(subject_class, invalid_subject_id, user, value)).to eq(false)
+    end
   end
 
   describe ".voted?" do
-   it "returns true if the user has voted" do
-    vote
-    expect(Vote.voted?(subject_class, subject_id, user)).to eq(true)
-    expect(Vote.voted?(subject_class, subject_id, user, value)).to eq(true)
-   end
+    it "returns true if the user has voted" do
+      vote
```

Comment:
Maybe add a `falsy` expectation to show it also returns false if the user has not voted
```ruby
it "returns true if the user has voted" do
expect(Vote.voted?(subject_class, subject_id, user)).to eq(false)
vote
:
:
end

```